attractions/table/table.scss

```diff
@@ -0,0 +1,76 @@
+@use 'sass:color';
+@use 'node_modules/attractions/_variables' as vars;
+
+table {
+  border-spacing: 0;
+  border-collapse: separate;
+  display: block;
+  overflow-x: auto;
+  margin-bottom: 1em;
+
+  th {
+    text-align: start;
+    padding: 0.5em 1em;
+
+    &.center {
+      text-align: center;
+    }
+
+    &.end {
+      text-align: end;
+    }
+  }
+
+  tr {
+    &.alternating {
+      &:nth-child(even) {
+        background: color.adjust(vars.$main, $alpha: -0.97);
+      }
+    }
+
+    &:first-child :global {
```

Comment:
Why are these styles global?

Reply:
idk. I copied them like that from the docs